.github/workflows/continuous-integration.yml

```diff
@@ -29,9 +29,15 @@ jobs:
         run: npm run build
 
       - name: test ${{ matrix.projects }} component
+        if: ${{ matrix.projects == 'server' }}
```

Comment:
Is this needed?

Reply:
Actually, let's change this in the project instead of in the pipeline

Reply:
Ah the server component can also run with the `npm run test --passWithNoTests` flag, so no i can merge them back into one